scss/bitstyles/settings/_layout.scss

```diff
@@ -1,6 +1,6 @@
 @use "sass:math";
 
-$bitstyles-global-z: topbar !default;
+$bitstyles-global-z: ('topbar') !default;
```

Comment:
Not sure I get that change 🙈

Reply:
Yeh it’s a drive-by really, something I’ve been trying to make better whenever I encounter it in the code — the original was a Sass list, but that wasn’t obvious because you don’t need brackets or commas to define a sass list 🙈 And the `topbar` doesn’t need quotes either.

I’ve been making this explicit every time I see it, so it’s clear when you’re dealing with a list of strings

Reply:
Perfect, thanks for the explanation 🙏